.github/workflows/upload-to-pypi.yml

```diff
@@ -0,0 +1,94 @@
+name: Publish release to PyPI
+
+on:
+  release:
+    types: [published]
```

Comment:
do you want to add a `prereleased` to this list to test this workflow/wheels with a 6.2.0 prerelease?

Reply:
Good question. I _think_ `published` covers pre-releases as well as releases. The documentation isn't 100% clear and leads you down a maze of twisty little passages, but I ended up here: https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#release

And that says:

> * published: a release, pre-release, or draft of a release is published

In addition, the main page https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release says:

> Note: The release event is not triggered for draft releases.

So I _think_ just `published` is what I need here.